modules/src/ics03_connection/exported.rs

```diff
@@ -42,6 +42,15 @@ impl State {
             Self::Open => "OPEN",
         }
     }
+
+    pub fn from_i32(nr: i32) -> Self {
```

Comment:
@ancazamfir was not sure this is accurate, please double-check.

Reply:
Looks good.